python/ray/exceptions.py

```diff
@@ -162,7 +162,13 @@ class RayTimeoutError(RayError):
     pass
 
 
+class PlasmaObjectNotAvailable(RayError):
```

Comment:
This may not be strictly necessary as it is only needed in Serialization.

Reply:
This is fine. `plasma.ObjectNotAvailable` doesn't appear in the codebase anywhere else so doesn't really matter what we return.